fe/fe-core/src/main/java/org/apache/doris/backup/RemoteFile.java

```diff
@@ -26,12 +26,14 @@ public class RemoteFile {
     private String name;
     private boolean isFile;
     private long size;
+    private long blockSize;
```

Comment:
Add comment to explain what is block size?

Reply:
Added